README.md

```diff
@@ -12,7 +12,7 @@ The last stable documentation is available for consultation at
 [mackwic.github.io/rspec](https://mackwic.github.io/rspec).
 
 **All rspec releases are garanteed to compile against the latest stable rust and
-are tested on all rust versions from the 1.9**.
+are tested on all rust versions from the 1.16**.
```

Comment:
Yes. I first went with 1.16, but then realized I need 1.18 for a thing I can't recall right now.